src/Analyzers/Core/Analyzers/IDEDiagnosticIdToOptionMappingHelper.cs

```diff
@@ -89,11 +89,7 @@ private static void AddOptionMapping(ConcurrentDictionary<string, ImmutableHashS
 #if DEBUG
             foreach (var option in options)
             {
-                if (option is IPerLanguageValuedOption)
-                {
-                    Debug.Assert(option.StorageLocations.OfType<RoamingProfileStorageLocation>().Single().GetKeyNameForLanguage(null).Contains("%LANGUAGE%"));
```

Comment:
Curious why this assert is removed?

Reply:
Oh I see the whole `RoamingProfileStorageLocation` is completely deleted now.

Reply:
Yup.